build.sbt

```diff
@@ -1038,10 +1038,17 @@ lazy val `sbt-plugin` = (project in file("dev") / "sbt-plugin")
     sbtPlugin := true,
     crossScalaVersions := Dependencies.SbtScalaVersions,
     scalaVersion := Dependencies.SbtScalaVersions.head,
+    sbtVersion in pluginCrossBuild := (CrossVersion.binaryScalaVersion(scalaVersion.value) match {
+      case "2.12" => "1.0.2"
+      case "2.11" => "0.13.16"
```

Comment:
Not sure why we need scala 2.11 here. I'd rather omit it so we get an error when we're dealing with an unexpected scala version.